src/source/reference/get-feed-empty.rst

```diff
@@ -1,7 +1,7 @@
 .. index:: GET /feed/empty
 .. _get_feed_empty:
 
-GET /feed/emtpy
+GET /feed/empty
```

Comment:
:D thanks for this one! Needed to go check the code if I made a typo there too :D